python/tests/core/test_base_table_scan.py

```diff
@@ -14,7 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
```

Comment:
Nit: this file doesn't need to change and could cause git conflicts.

Reply:
agreed, fixed